web-client/integration-tests-public/unauthedUserCaseSearchExactMatchMultipleFields.test.ts

```diff
@@ -21,7 +21,7 @@ const [firstName, lastName] = [
 
 const nameToSearchFor = `${firstName} ${lastName}`;
 
-const createdDocketNumbers = [];
+const createdDocketNumbers: any[] = [];
```

Comment:
small nit: I think it's safe to assume it'll be an array of strings in this instance. It may be more precise to type as such.